query_optimizer/LogicalGenerator.cpp

```diff
@@ -43,6 +45,7 @@ LogicalGenerator::~LogicalGenerator() {}
 L::LogicalPtr LogicalGenerator::generatePlan(
     const ParseStatement &parse_statement) {
   resolver::Resolver resolver(optimizer_context_);
+  DVLOG(4) << "Parse tree:\n" << parse_statement.toString();
```

Comment:
I guess, this maybe a residual print statement from debugging. Wasn't earlier, unless we want to print the parse tree now too.